sdk/eventgrid/azure-eventgrid/tests/test_eg_publisher_client.py

```diff
@@ -75,11 +109,26 @@ def test_eg_publisher_client_publish_cloud_event_data_str(self, resource_group,
                 type="Sample.Cloud.Event"
                 )
         client.send([cloud_event])
-    
+
+    @pytest.mark.liveTest
+    @CachedResourceGroupPreparer(name_prefix='eventgridtest')
+    @CachedEventGridTopicPreparer(name_prefix='cloudeventgridtest')
+    def test_send_cloud_event_dict(self, resource_group, eventgrid_topic, eventgrid_topic_primary_key, eventgrid_topic_endpoint):
```

Comment:
if we're changing to normalize as "test_eg_publisher_client_publish" let's adjust this accordingly.

(Truthfully I think we could have kept it terse with test_publish_*, e.g. test_publish_cloud_event_dict)